Modules/TPC/include/TPC/SliceInfo.h

```diff
@@ -85,11 +86,17 @@ struct SliceInfo {
     }
   }
 
+  std::string RetrieveTitle() const
+  {
+    return title;
+  }
+
```

Comment:
Why do you need this function? `title` is public in the struct.
```suggestion
```

Reply:
Changed as requested